pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -18,6 +18,7 @@
 from typing import Dict, List, Optional, Union
 
 import torch
+from typing_extensions import Literal
```

Comment:
jsonargparse currently does not consider backports in `typing_extensions`. Could be added.

Reply:
It is now added, [jsonargparse/commit/16cec7e](https://github.com/omni-us/jsonargparse/commit/16cec7e8a6aff99be5995eafa232b6e3b5c46dc2). Will be part of the next release.